quill/test/HandlerCollectionTest.cpp

```diff
@@ -121,16 +129,20 @@ TEST_CASE("subscribe_get_active_different_handlers")
 
 #if defined(_WIN32)
   // Create a file handler
-  StreamHandler* filehandler = hc.file_handler(L"create_get_file_handler_1", "w");
+  StreamHandler* filehandler =
+    hc.create_handler<FileHandler>("create_get_file_handler_1", "w", FilenameAppend::None);
```

Comment:
@odygrd Looks like the `L` was removed here too.
```suggestion
    hc.create_handler<FileHandler>(L"create_get_file_handler_1", "w", FilenameAppend::None);
```